tests/versioned/apollo-server/attributes.test.js

```diff
@@ -9,7 +9,7 @@ const test = require('node:test')
 
 const { afterEach, setupCoreTest } = require('../../lib/test-tools')
 
-const attributesTestSuite = require('../attributes-tests')
+const attributesTestSuite = require('./attributes-tests')
```

Comment:
i don't see the need for this file. you can prob just move the `attributes-tests` into here or make that file run the tests.